ios/Plugin/Banner/BannerExecutor.swift

```diff
@@ -88,9 +88,17 @@ class BannerExecutor: NSObject, GADBannerViewDelegate {
                 NSLog("AdMob: find subView for resumeBanner")
                 subView.isHidden = false
 
+                let width = subView.frame.width
+                var height = subView.frame.height
+                // For fixing issue 160 iOs banner overlapping content on iPhone with rounded corners                                
+                if let window = UIApplication.shared.keyWindow {
+                    let safeAreaBottom = window.safeAreaInsets.bottom
+                    height = height + safeAreaBottom
+                }
```

Comment:
Can you please do not change outside context inside the curly braces?

You can do it with something like
```
let safeAreaBottom = UIApplication.shared.keyWindow?.safeAreaInsets.bottom ?? 0
height += safeAreaBottom
```